JavaShared/src/main/java/org/openzen/zenscript/javashared/compiling/JavaCompilingClass.java

```diff
@@ -125,29 +125,33 @@ public void addMethod(MethodSymbol method, NativeTag native_) {
 		if (native_ != null && nativeClass != null) {
 			final String signature = getContext().getMethodSignature(method.getHeader());
 			JavaMethod method1 = nativeClass.getMethod(native_.value);
-			addMethod(method, method1.asCompilingMethod(compiled, signature), method1);
-		} else {
-			final JavaNativeMethod.Kind kind = getKind(method);
-			final String descriptor;
-			final String signature;
-			if (kind == JavaNativeMethod.Kind.EXPANSION) {
-				descriptor = getContext().getMethodDescriptorExpansion(method.getHeader(), method.getTargetType());
-				signature = getContext().getMethodSignatureExpansion(method.getHeader(), method.getTargetType());
-			} else {
-				descriptor = getContext().getMethodDescriptor(method.getHeader());
-				signature = getContext().getMethodSignature(method.getHeader());
+			if (method1 != null) {
```

Comment:
If a method has `[Native]` in the ZC file, but the JavaPrepareDefinitionVisitor does not add the member, then we NPE here.
This change makes is so that we compile the method normally in that case.
Is this the proper place to make that change (or should the `native_` variable have already been cleared to `null` beforehand in these cases?